frontend/public/components/masthead-toolbar.jsx

```diff
@@ -378,8 +393,22 @@ class MastheadToolbarContents_ extends React.Component {
       ],
     });
 
-    if (!_.isEmpty(additionalHelpActions.actions)) {
-      helpActions.push(additionalHelpActions);
+    const additionOperatorHelpActions = { ...additionalHelpActions } || {
```

Comment:
Won't the first condition always evaluate as truthy?

Reply:
Yep - this was an artifact of a previous version of the code.